packages/salesforcedx-vscode-apex/src/commands/forceLaunchReplayDebugger.ts

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import {
+  EmptyParametersGatherer,
+  SfdxCommandlet,
+  SfdxCommandletExecutor,
+  SfdxWorkspaceChecker
+} from '@salesforce/salesforcedx-utils-vscode/out/src';
+import {
+  Command,
+  CommandBuilder
+} from '@salesforce/salesforcedx-utils-vscode/out/src/cli';
+import {
+  notificationService
+} from '@salesforce/salesforcedx-utils-vscode/out/src/commands';
+import * as vscode from 'vscode';
+import { nls } from '../messages';
+import {
+  testOutlineProvider
+} from '../views/testOutlineProvider';
+import {
+  forceAnonApexDebug
+} from './forceApexExecute';
+
+export async function forceLaunchReplayDebugger() {
+  const editor = vscode.window.activeTextEditor;
+  if (!editor) {
+    notificationService.showErrorMessage(
+      nls.localize('unable_to_locate_editor')
```

Comment:
I'm working with Sonal on the strings on lines 34, 42, and 59.

Reply:
"You can run this command only on a source file."
"You can run this command only on an Anonymous Apex file, or an Apex test file."